### DISCUSSION TITLE
Motion Detect in photo or burst mode

### DISCUSSION THREAD

**Discussion by RedLefty** [gopro/labs#953] [Q&A]
Hi- Are these functions feasible or likely in the future?

**Comment by dnewman-gpsw** [comment]
Which camera model?

**Comment by RedLefty** [comment]
I have both 8 Black and 11 Black

**Reply by dnewman-gpsw** [reply]
8 - No.
11 - Maybe future.

11 and later support the new Blockly visual scripting, which also has a different motion detection control.  You could experiment with that for photos. https://gopro.github.io/labs/build/

However it wasn't designed for photos.  It is being improved, but those enhancements are not out yet.

**Comment by RedLefty** [comment]
Many thanks - development work is way above my pay grade!
